basis-universal-sys/vendor/transcoding_wrapper.cpp

```diff
@@ -157,7 +157,32 @@ extern "C" {
     // basisu_lowlevel_uastc_transcoder
     //
 
-    // Not implemented
+    struct LowLevelUastcTranscoder {
+        basist::basisu_lowlevel_uastc_transcoder *pTranscoder;
+    };
+
+    LowLevelUastcTranscoder *low_level_uastc_transcoder_new() {
+        LowLevelUastcTranscoder *transcoder = new LowLevelUastcTranscoder;
+        transcoder->pTranscoder = new basist::basisu_lowlevel_uastc_transcoder();
+        return transcoder;
+    };
+
+    bool low_level_uastc_transcoder_transcode_slice(LowLevelUastcTranscoder *transcoder, void* pDst_blocks,
```

Comment:
Since there's so many parameters here, could you please format this similar to basis_validate_output_buffer_size() in line 128 where each param is on its own line?

Reply:
I would love to! :) Done.